src/pages/strategies/create/overlapping.test.tsx

```diff
@@ -0,0 +1,70 @@
+import { test, expect, afterEach, vitest } from 'vitest';
+import { renderWithRouter, screen } from 'libs/testing-library';
+import { debugTokens } from '../../../../e2e/utils/types';
+import { CreateOverlappingStrategyPage } from './overlapping';
+import userEvent from '@testing-library/user-event';
+
+vitest.mock('hooks/useMarketPrice', () => {
+  return {
+    useMarketPrice: () => {
+      return { marketPrice: 2800, isPending: false };
+    },
+  };
+});
+
+const basePath = '/strategies/create/overlapping';
+afterEach(async () => {
+  window.history.replaceState(null, 'root', '/');
+});
+
+test('run overlapping form with deeplink and user market price', async () => {
+  // Initialize user and set params
+  const user = userEvent.setup();
+  const search = {
```

Comment:
Ideally you would use typescript interface to make sure we update there if there is a change in the interface in the future